lib/build.gradle

```diff
@@ -60,3 +63,90 @@ dependencies {
     testCompile 'org.mockito:mockito-core:2.7.17'
     testCompile 'org.robolectric:robolectric:3.3.1'
 }
+
+def siteUrl = 'https://github.com/willowtreeapps/spruce-android'
+def gitUrl = 'https://github.com/willowtreeapps/spruce-android.git'
+group = "com.willowtreeapps.spruce"
+version = "1.0.0"
+
+install {
+    repositories.mavenInstaller {
+        // This generates POM.xml with proper parameters
+        pom {
+            project {
+                packaging 'aar'
+
+                name 'spruce-android'
+                description = 'lightweight animation library'
+                url siteUrl
+
+                licenses {
+                    license {
+                        name 'MIT License'
+                        url 'https://opensource.org/licenses/MIT'
+                    }
+                }
+                developers {
+                    developer {
+                        id 'willowtreeapps'
+                        name 'WillowTree Inc'
+                        email 'developer@willowtreeapps.com'
+                    }
+                }
+                scm {
+                    connection gitUrl
+                    developerConnection gitUrl
+                    url siteUrl
+                }
+            }
+        }
+    }
+}
+
+task sourcesJar(type: Jar) {
+    from android.sourceSets.main.java.srcDirs
+    classifier = 'sources'
+}
+
+task javadoc(type: Javadoc) {
+    source = android.sourceSets.main.java.srcDirs
+    classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
+}
+
+task javadocJar(type: Jar, dependsOn: javadoc) {
+    classifier = 'javadoc'
+    from javadoc.destinationDir
+}
+
+artifacts {
+    archives javadocJar
+    archives sourcesJar
+}
+// Used for local deployments
+//Properties properties = new Properties()
+//properties.load(project.rootProject.file('local.properties').newDataInputStream())
+//
+//bintray {
```

Comment:
it's pulling from local.properties which is ignored for configuration purposes. I'd rather it not be commented but I don't wanna have a stub local.properties either.